src/seedu/addressbook/commands/Command.java

```diff
@@ -63,6 +63,10 @@ protected ReadOnlyPerson getTargetPerson() throws IndexOutOfBoundsException {
         return relevantPersons.get(getTargetIndex() - DISPLAYED_INDEX_OFFSET);
     }
 
+    public boolean isMutating() {
```

Comment:
This method should have some comments as it will be overridden later. This is to inform others how it should behave.